config/katello.migrations/200123161606-enable-pulpcore.rb

```diff
@@ -0,0 +1,18 @@
+PUPPET_CLASS = 'foreman_proxy::plugin::pulp'
+PARAM = 'pulpcore_enabled'
+
+def upgrade
+  answers[PUPPET_CLASS][PARAM] = true
+  return unless answers['katello']
```

Comment:
This should be a block now (and indent the rest):
```suggestion
  unless answers['katello']
```